RNUnifiedContacts/RNUnifiedContacts.swift

```diff
@@ -163,17 +163,94 @@ class RNUnifiedContacts: NSObject {
 
       callback([error.localizedDescription, NSNull()])
     }
-
   }
-
-
-
-
-
+  
+  @objc func createContact(contactData: NSDictionary, callback: (NSObject) -> () ) -> Void {
+    
+    let mutableContact = CNMutableContact()
+    let contactStore = CNContactStore()
+    let req = CNSaveRequest()
+    
+    // TODO:
+    // Extend method to handle more fields
+    //
+    //
+    mutableContact.givenName = contactData["givenName"] as! String
+    mutableContact.familyName = contactData["familyName"] as! String
+    mutableContact.organizationName = contactData["organizationName"] as! String
+    
+    for number in contactData["phoneNumbers"] as! NSArray {
+      let transformedNumber = transformNumber( number as! NSDictionary )
```

Comment:
`transformedNumber` and `transformNumber` don't provide much indication of what it is being transformed **in to** (i.e. what's being returned). 

Instead, what about calling the function `convertNumberToNSDictionary` and the variable `numberAsNSDictionary`?